<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChifoumiVue</class>
 <widget class="QMainWindow" name="ChifoumiVue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChifoumiVue</string>
  </property>
  <widget class="QWidget" name="fenetrePrincipalH">
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>450</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="fenetreSecondaireV">
      <item>
       <layout class="QGridLayout" name="grilleTop">
        <item row="0" column="2">
         <layout class="QVBoxLayout" name="vLayout_MachineScore">
          <item>
           <widget class="QLabel" name="lMachine">
            <property name="text">
             <string>Machine</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lScoreMachine">
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="vLayout_JoueurScore">
          <item>
           <widget class="QLabel" name="label_Vous">
            <property name="text">
             <string>Vous</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lScoreJoueur">
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="hLayout_ChoixJoueur">
          <item>
           <spacer name="hSpacerG_ChoixJoueur">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lChoixJoueur">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>110</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>110</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="ressourceChifoumi.qrc">:/chifoumi/images/rien.gif</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hSpacerD_ChoixJoueur">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="hLayout_Versus">
          <item>
           <spacer name="hSpacerG_Versus">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lVersus">
            <property name="minimumSize">
             <size>
              <width>78</width>
              <height>78</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>78</width>
              <height>78</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="ressourceChifoumi.qrc">:/chifoumi/images/versus_70.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hSpacerD_Versus">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <layout class="QHBoxLayout" name="hLayout_ChoixMachine">
          <item>
           <spacer name="hSpacerG_ChoixMachine">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lChoixMachine">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>110</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>110</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="ressourceChifoumi.qrc">:/chifoumi/images/rien.gif</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hSpacerD_ChoixMachine">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="vSpacerMiddle">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Choisissez une figure :</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="hSpacerG_GroupBox">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="boutonPierre">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="ressourceChifoumi.qrc">
             <normaloff>:/chifoumi/images/pierre.gif</normaloff>:/chifoumi/images/pierre.gif</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="boutonPapier">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="ressourceChifoumi.qrc">
             <normaloff>:/chifoumi/images/papier.gif</normaloff>:/chifoumi/images/papier.gif</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="boutonCiseau">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="ressourceChifoumi.qrc">
             <normaloff>:/chifoumi/images/ciseau.gif</normaloff>:/chifoumi/images/ciseau.gif</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hSpacerD_GroupBox">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="grilleBottom">
        <item row="1" column="1">
         <widget class="QPushButton" name="boutonNvllePartie">
          <property name="text">
           <string>Nouvelle partie</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="hSpacerNvllePartie">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAPropos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAPropos">
   <property name="text">
    <string>A Propos de</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressourceChifoumi.qrc"/>
 </resources>
 <connections/>
</ui>
